fastlane/Fastfile

```diff
@@ -536,22 +536,39 @@ platform :ios do
     }
   end
 
-  desc "Export XCFramework"
-  lane :export_xcframework do |options|
-    output_directory = 'build/xcframeworks'
-    platforms = [
-        'iOS',
-        'macOS',
-        'maccatalyst',
-        'tvOS',
-        'watchOS',
-        'visionOS'
-      ]
+  lane :export_xcframeworks do |options|
+    create_and_sign_xcframework(
+      scheme: 'RevenueCat',
+      product_name: 'RevenueCat',
+      platforms: [
+      'iOS',
+      'macOS',
+      'maccatalyst',
+      'tvOS',
+      'watchOS',
+      'visionOS'
+    ])
+    create_and_sign_xcframework(
+      scheme: 'RevenueCatUIDev',
+      product_name: 'RevenueCatUI',
+      platforms: [
+      'iOS',
+      'maccatalyst',
+      'watchOS',
+    ])
+  end
+
+  private_lane :create_and_sign_xcframework do |options|
+    scheme = options[:scheme]
+    product_name = options[:product_name]
+    platforms = options[:platforms]
+
+    output_directory = "build/xcframeworks/#{product_name}"
```

Comment:
Might be totally not applicable, but should we upload this as an artifact on CircleCI?

Reply:
not a bad idea, might be useful for the release checks lane. I'll do it as a separate thing, though, since we haven't done it before

Reply:
Sounds good!

Reply:
actually it's a brilliant idea, not sure why I said "not bad" 😅 I'm tackling it here: https://github.com/RevenueCat/purchases-ios/pull/4189

Reply:
Haha, it's the greatest midwestern compliment, apparently. Or was that "I don't hate it"? 😄 Anyway, awesome!